pkgs/development/python-modules/playwright/default.nix

```diff
@@ -46,10 +46,13 @@ buildPythonPackage rec {
     git commit -m "workaround setuptools-scm"
 
     substituteInPlace setup.py \
-      --replace "greenlet==2.0.1" "greenlet>=2.0.1" \
-      --replace "pyee==8.1.0" "pyee>=8.1.0" \
-      --replace "setuptools-scm==7.0.5" "setuptools-scm>=7.0.5" \
-      --replace "wheel==0.38.1" "wheel>=0.37.1"
+      --replace "greenlet==3.0.1" "greenlet>=3.0.1" \
+      --replace "pyee==11.0.1" "pyee>=11.0.1" \
+      --replace "setuptools-scm==8.0.4" "setuptools-scm" \
+      --replace "wheel==0.41.2" "wheel"
```

Comment:
Use `pythonRelaxDepsHook` instead.

Reply:
Please correct me if I am wrong, but I believe that I could do so only for greenlet and pyee as pythonRelaxDepsHook only affects the built wheels Requires-Dist, but setup.py defines the other two in setup_requires.

And it gets worse with pyproject = true, as upstream [abuses build-system.requires](https://github.com/microsoft/playwright-python/blob/85df32c24e4d887f65fa0f1421d3d90c716fd90a/pyproject.toml#L2) there which is also unaffected by pythonRelaxDepsHook. I am patching this out in the latest iteration. Fixing this upstream would probably be ideal but I don't feel like finding out whether signing microsofts CLA linked in Contributing.md is required for such a change ;)

Reply:
Yes, this only works for runtime dependencies, which are specified in the `Require-Dist metadata fields.

Migrating to PEP517 builds is something that needs to happen at some point down the road, so I'd rather we do it now than later.

Reply:
I just did, see my latest push. Just still patching via substituteInPlace, not pythonRelaxDepsHook.

Reply:
Fine with me.